src/run.rs

```diff
@@ -50,8 +50,15 @@ pub async fn image_ids(docker: &Docker) -> io::Result<HashSet<String>> {
                 format!("Unable to determine IDs of all images: {:?}.", error),
             )
         })?;
+    let mut images = HashMap::new();
 
-    Ok(output.into_iter().map(|image| image.id).collect())
+    for image in output {
+        match image.created.try_into() {
+            Ok(unsigned) => images.insert(image.id, Duration::from_secs(unsigned)),
+            Err(e) => return Err(io::Error::new(io::ErrorKind::Other, e))
```

Comment:
Elsewhere in this file we use the name `error` rather than `e` when pattern matching on an `Err`. Can we do that here too? (Alternatively we could change all the other ones. I don't feel strongly about it, as long as we're consistent.)